src/components/modals/AddDulModal.js

```diff
@@ -184,136 +185,136 @@ export const AddDulModal = hh(class AddDulModal extends Component {
   render() {
 
     const file = {
-      name: "MyFile.txt"
+      name: 'MyFile.txt'
     };
 
     return (
 
       BaseModal({
-        id: "addDulModal",
-        disableOkBtn: this.state.file === '' || this.state.disableOkBtn,
-        showModal: this.props.showModal,
-        onRequestClose: this.closeHandler,
-        onAfterOpen: this.afterOpenHandler,
-        imgSrc: "/images/icon_add_dul.png",
-        color: "dul",
-        title: this.state.isEditMode ? "Edit Data Use Limitations" : "Add Data Use Limitations",
-        description: this.state.isEditMode ? "Edit a Data Use Limitations Record" : "Catalog a Data Use Limitation Record",
-        action: {
-          label: this.state.isEditMode ? "Edit" : "Add",
-          handler: this.OKHandler
-        }
-      },
+          id: 'addDulModal',
+          disableOkBtn: this.state.file === '' || this.state.disableOkBtn,
+          showModal: this.props.showModal,
+          onRequestClose: this.closeHandler,
+          onAfterOpen: this.afterOpenHandler,
+          imgSrc: this.state.isEditMode ? '/images/icon_edit_dul.png' : '/images/icon_add_dul.png',
```

Comment:
This is the only substantive change ... everything else is pure eslint formatting fixes.